x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/parser/AbstractStatementParserTests.java

```diff
@@ -140,22 +140,30 @@ static MapExpression mapExpression(Map<String, Object> keyValuePairs) {
     }
 
     void expectError(String query, String errorMessage) {
```

Comment:
This could just defer to `expectError(String query, List<QueryParam> params, String errorMessage)` with `params` given by ` new QueryParams()`.